packages/ant-component-mapper/src/files/form-template.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { Button, Typography } from 'antd';
+import 'antd/dist/antd.css';
```

Comment:
Importing of styles should be handled by users in their application. You can remove it here.